polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -317,7 +317,9 @@ export class CellContainer extends Disposable {
             dragHandle.disabled = state.running || state.queued;
         })
 
-        this.el = div(['cell-and-divider'], [
+        // If code cell, check if the cell container should be rendered in a side display
+        const cellAndDividerClasses = ['cell-and-divider', ... (this._cell.splitDisplay) ? ['split-display-container'] : []];
```

Comment:
It's a bit annoying to have to introduce logic about the cell type here since we already have this logic in the `CodeCell` class, but the `CodeCell` is created before the `CellContainer` is, so this check has to be here. Not sure if better way to do this